### DISCUSSION TITLE
Forked repository skipped with "includeForks": true in config

### DISCUSSION THREAD

**Discussion by Chr1stian** [renovatebot/renovate#9738] [Request Help]
**What Renovate type, platform and version are you using?**

"renovateVersion": "24.104.0" 
Self-hosted BitBucket Cloud

**Describe the bug**

After update to 24.104.0 and using the new shared local config preset for BitBucket Cloud a previously working repository now gets skipped because it is a forked repo.

**Relevant debug logs**

<details><summary>Click me to see logs</summary>

```
2021-04-26 10:59:10.808 CEST
"durationMs": 2380
Info
2021-04-26 10:59:10.808 CEST
INFO: Repository finished (repository=organization/repo-name)
Info
2021-04-26 10:59:10.806 CEST
INFO: Repository is a fork and not manually configured - skipping (repository=organization/repo-name)
Info
2021-04-26 10:59:08.429 CEST
"renovateVersion": "24.104.0" 
```

</details>

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ X ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

I have tried setting "includeForks": true in only the repo config, only the shared local preset config and both. It does not seem to register. All other shared options seem to work fine for the other repositories that are not forks.

**Comment by viceice** [comment]
`includeForks`  needs to be in app `config.js` or repo `renovate.json`

**Reply by rarkins** [reply]
To clarify: when a Repository is a fork, Renovate doesn't check all the possible config files in the repo - only `renovate.json`.

If you want all forks to be Renovated then set `"includeForks": true`  in `config.js` or `--include-forks=true` in CLI or `RENOVATE_INCLUDE_FORKS=true` in env.

**Reply by viceice** [reply]
https://github.com/renovatebot/renovate/blob/b957d4afb7824a165864cbd6019a7540b315ebe9/lib/workers/repository/configured.ts#L7-L14

https://github.com/renovatebot/renovate/blob/2cc751a0a1ba2e0d3bc35f51eed4cdaf07f8282c/lib/workers/repository/init/index.ts#L31-L32

**Reply by viceice** [reply]
https://github.com/renovatebot/renovate/blob/b957d4afb7824a165864cbd6019a7540b315ebe9/lib/workers/repository/onboarding/branch/index.ts#L25-L27

`includeForks` must be in one of the repo suported config files.

**Reply by viceice** [reply]
Supported files:
https://github.com/renovatebot/renovate/blob/0b5ee0b40083644445cd5b42696d3f7eff94760a/lib/config/app-strings.ts#L1-L11

**Reply by kitos9112** [reply]
Aham! thanks for these comments here it just solved my particular issue in a [forked public Github repo](https://github.com/kitos9112/traefik-forward-auth) that I'm maintaining at the moment. I couldn't find this documented [anywhere ](https://docs.renovatebot.com/configuration-options/#includeforks) except here.

Are there any other specific repo-settings that can only be set from a `renovate.json` file at the root of the repository?
